# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sbc/hotfix-non-x86.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sbc-1.5/sbc/sbc_primitives.c.vanilla	2021-10-06 13:40:46.357688186 +0200
+++ sbc-1.5/sbc/sbc_primitives.c	2021-10-06 13:42:01.580685187 +0200
@@ -593,7 +593,9 @@
 
 static void sbc_init_primitives_x86(struct sbc_encoder_state *state)
 {
+#if SBC_BUILD_WITH_MMX_SUPPORT == 1 || SBC_BUILD_WITH_MMX_SUPPORT == 1
 	__builtin_cpu_init();
+#endif
 
 #ifdef SBC_BUILD_WITH_MMX_SUPPORT
 	if (__builtin_cpu_supports("mmx"))
